CodeGeneration/Sources/SyntaxSupport/ExprNodes.swift

```diff
@@ -1676,7 +1686,8 @@ public let EXPR_NODES: [Node] = [
         kind: .token(choices: [.keyword(text: "switch")])
       ),
       Child(
-        name: "Expression",
+        name: "Subject",
```

Comment:
Reference calls this the "control expression" FWIW

Reply:
The compiler calls it “subject”. I think I prefer “subject”.